src/components/DashBoard.vue

```diff
@@ -216,50 +252,48 @@ body.sb-sidenav-toggled #wrapper #sidebar-wrapper {
 }
 
 .dashboard {
-  color: #FA6731;
+  color: #fa6731;
   letter-spacing: 0.15px;
   line-height: 6px;
   margin-top: -1px;
   min-width: 42px;
   white-space: nowrap;
-  
 }
 
-.dashboard-menu{
-  border:none !important;
-  padding:15px !important;
-  text-decoration:none;
-  color:black;
+.dashboard-menu {
+  border: none !important;
+  padding: 15px !important;
+  text-decoration: none;
+  color: black;
 }
-.navbar-account-img{
-  height:35px;
-  width:35px;
-  border:1px solid grey;
+.navbar-account-img {
+  height: 35px;
+  width: 35px;
+  border: 1px solid grey;
   border-radius: 50%;
-
 }
-.nav-link{
-  position:relative !important;
+.nav-link {
+  position: relative !important;
 }
-.notification-count{
-  height:15px;
-  width:15px;
-  border-radius:50%;
+.notification-count {
+  height: 15px;
+  width: 15px;
+  border-radius: 50%;
   background-color: red;
-  position:absolute;
-  right:0;
-  top:0;
-  color:white;
+  position: absolute;
+  right: 0;
+  top: 0;
+  color: white;
   text-align: center;
-  font-size:10px;
+  font-size: 10px;
 }
 /* .dashboard-menu:hover {
  color: #FA6731 ; 
 } */
-.router-link-exact-active{
-  border-radius:5px !important;
-  color:white !important;
-  background-color: #FA6731 ;
+.router-link-exact-active {
+  border-radius: 5px !important;
```

Comment:
Why is it required to add "!important"? Can't we remove it?

Reply:
We're adding !important property to override the predefined bootstrap class properties